templates/home/_fsf_rust.html

```diff
@@ -10,8 +10,45 @@ <h4>Why are snaps good for Rust projects?</h4>
     </ul>
```

Comment:
The above needs updating. Please use the Python flow as a guide and add a paragraph about how snaps compare to / fit alongside existing rust tooling, i.e. cargo.

Reply:
Have a look at the Go text. They're similar languages from a project management perspective.


> * Easy to discover and install by millions using the Snap Store or command-line every day
> * Automatically updated to the latest stable version of your app
> * Revert to the previous version if an update fails, preserving data
>
> Programming in Go makes it easy to create a zip of your app that runs across Linux, without dependencies. However, end user discovery and update management remain a challenge. Snaps fill this gap, letting you distribute a Go app in an app store experience for end users.